File: codeql-custom-queries-java/android/AvoidKeepScreenOn.java
package android.view;

class WindowManager {
    public static class LayoutParams {
        public static final int FLAG_KEEP_SCREEN_ON = 128;
    }
}

public class AvoidKeepScreenOn {
    void success() {
        int x = WindowManager.LayoutParams.FLAG_KEEP_SCREEN_ON;
    }
}

File: codeql-custom-queries-java/android/avoid-keep-screen-on.kt
package android.view

class WindowManager {
    class LayoutParams {
        companion object {
            const val FLAG_KEEP_SCREEN_ON = 128
        }
    }
}

class AvoidKeepScreenOn {
    fun success() {
        val x = WindowManager.LayoutParams.FLAG_KEEP_SCREEN_ON // $ Alert
    }
}